api/migrations/0006_auto_20181127_0004.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-27 00:04
+from __future__ import unicode_literals
+
+import api.models
+import custom_storages
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0005_astronauts_orbiter_orbiterflight_spacestation'),
```

Comment:
This is still wrong, migrations have to be linear - the last migration on master/dev was 0006_astronauts_orbiter_orbiterflight_spacestation, therefore the next migration should start with 0007_ and should depend on the previous.